index.html

```diff
@@ -63,7 +63,110 @@ <h4>Skills</h4>
             <a href="#"><img alt= 'Twitter' src="./assets/Twitter.svg"></a>
             <a href="#"><img alt= 'M' src="./assets/M.svg"></a>
         </div>
-        <hr>                    
+        <span>&#8595; &nbsp;&nbsp; WORK</span>
+        <div class="works-flex">
+            <h1>MY RECENT WORK</h1>
+            <br>
+            <hr>
+            
+            <div class="project-flex">
+                <div class="content-flex">
```

Comment:
Your project cards don't quite look like those from the template design. Please refer to the Figma template and make sure the layout is identical to the template design, particularly when it comes to the list of technologies.

<img width="482" alt="Screen Shot 2022-07-07 at 21 46 49" src="https://user-images.githubusercontent.com/82426682/177906861-a9c4e83d-3fd5-46eb-a4ed-7d31d477a119.png">

Reply:
The margins have been corrected according to the design.